### ISSUE TITLE
Make the game responsive

### ISSUE THREAD

**Issue by mizanxali** [mizanxali/uno-online#2]
The game isn't optimized for mobile screens right now. Use CSS media queries to change the size of the game elements to adjust themselves to mobile screen dimensions.

**Comment by programmerraja** [comment]
Bro  can I work on this ?

**Comment by mizanxali** [comment]
> Bro can I work on this ?

Yes, you can! Make sure to follow the contribution instructions in the README.

**Comment by programmerraja** [comment]
Ok bro

**Comment by programmerraja** [comment]
Hai bro I just like to inform you that I added media queries for home page and for spinner page 
I only need to add is game page but it's little bit difficult to do but I will do it bro ✌️

**Comment by programmerraja** [comment]
Bro at last I did it but it look good in mobile may be not responsive in tab 
Did I convert it responsive for tab also ?
Or did I pull my request ?
